simulation/deploy_contracts.py

```diff
@@ -11,21 +12,17 @@
 
 # web3.py instance
 w3 = Web3(HTTPProvider('http://127.0.0.1:8545'))
+w3.personal.unlockAccount(w3.eth.accounts[0], '1')
 
 ESCROW_ADDRESS = w3.eth.accounts[1]
 HOLDERS_ACCOUNTS = w3.eth.accounts[1:4]
```

Comment:
in this place is better than the number 4 to replace on a constant ACCOUNT_INDENT